server/routes/flashcard.js

```diff
@@ -1,9 +1,16 @@
-// import express from "express";
-// import { signIn, signUp } from "../controllers/user.js";
+import express from "express";
+import * as flashcardControllers from "../controllers/flashcard.js";
 
-// const router = express.Router();
+const flashcardRouter = express.Router();
 
-// router.post("/signIn", signIn);
-// router.post("/signUp", signUp);
+flashcardRouter.post("/", flashcardControllers.createCard);
+flashcardRouter.get("/:id", flashcardControllers.getCard);
+flashcardRouter.get("/", flashcardControllers.getCards);
```

Comment:
maybe I missed it but what's the path to flashcards? it shouldn't be for ex `amgi.com/`, shouldn't it be something along the line of `amgi.com/deck/flashcards/:id`? Or is this just for testing purposes for now? Consider adding `app.use("/deck/<your route>", flashcardRoute);` in `index.js`